.github/workflows/ci.yml

```diff
@@ -75,10 +75,11 @@ jobs:
         uses: mozilla-actions/sccache-action@v0.0.4
       - run: rustup update stable && rustup default stable
       - run: make build-xcframework
-      - name: Select Xcode 15.4
-        run: sudo xcode-select -s /Applications/Xcode_15.4.app
-      - name: Build ${{ matrix.config }}
-        run: make CONFIG=${{ matrix.config }} build-swift-apple-platforms
+      # So confused on what to run for Swift CI
+      # - name: Select Xcode 15.4
+      #   run: sudo xcode-select -s /Applications/Xcode_15.4.app
+      # - name: Build ${{ matrix.config }}
+      #   run: make CONFIG=${{ matrix.config }} build-swift-apple-platforms
```

Comment:
we discussed that this should compile after running `make build-xcframework`

Reply:
It doesn't, but `make build-xcframework` works. Seems this issue in somewhere Swift-side vs this code